examples/third-party/haproxy-ingress/20_grafana.cm.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-datasources
+data:
+  prometheus.yaml: |-
+    {
+        "apiVersion": 1,
+        "datasources": [
+            {
+               "access":"proxy",
+                "editable": true,
+                "name": "prometheus",
+                "orgId": 1,
+                "type": "prometheus",
+                "url": "http://prometheus.haproxy-ingress.svc:9090",
```

```suggestion
                "url": "http://prometheus:9090",
```
(so it works even in different namespaces)